test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -113,6 +113,7 @@ private HapiApiSpec canGetBalanceAndInfoViaAlias() {
 								.payingWith(GENESIS)
 								.via(autoCreation)
 				).then(
+						getTxnRecord(autoCreation).andAllChildRecords().logged(),
```

Comment:
Should we validate the child record has `parentConsensusTime` ?

Reply:
In this case the child record **won't** have a `parentConsensusTime`, because the auto-create record comes before the top-level `CryptoTransfer`.